src/SFA.DAS.AAN.Hub.Data/Repositories/JobAuditRepository.cs

```
@@ -0,0 +1,50 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.AAN.Hub.Data.Entities;
+using SFA.DAS.AAN.Hub.Data.Interfaces;
+using System;
+
+namespace SFA.DAS.AAN.Hub.Data.Repositories
+{
+    public class JobAuditRepository : IJobAuditRepository
+    {
+        private readonly ILogger<JobAuditRepository> _logger;
+        private readonly IAanDataContext _context;
+
+        public JobAuditRepository(ILogger<JobAuditRepository> logger, IAanDataContext context) 
+        { 
+            _logger = logger;
+            _context = context;
+        }
+
+        public async Task<JobAudit?> GetMostRecentJobAudit(CancellationToken cancellationToken)
+        {
+            try
```

I dont like suppressing errors, we are not expecting this operations to fail and hence we should not be catching any errors, this will allow the actual exceptions to bubble up and Application Insights can then show us exactly where it occurred. Please remove all the instances, across files, where try catches are suppressing exceptions.